src/irmin/irmin.mli

```diff
@@ -1727,18 +1727,22 @@ module Private : sig
           operations. *)
 
       val closure :
-        [> `Read ] t -> min:node list -> max:node list -> node list Lwt.t
-      (** [closure t ~min ~max] is the transitive closure [c] of [t]'s
-          nodes such that:
+        [> `Read ] t ->
+        min:node list ->
+        max:node list ->
+        (node list, [ `Disconnected of node | `Empty_max ]) result Lwt.t
+      (** [closure t ~min ~max] is the list of nodes [n] reachable from a
+          node of [max] along a path which : (i) either contains no [min] or
+          (ii) it ends with a [min].
 
           {ul
-          {- There is a path in [t] from any nodes in [min] to nodes
-          in [c]. If [min] is empty, that condition is always true.}
-          {- There is a path in [t] from any nodes in [c] to nodes in
-          [max]. If [max] is empty, that condition is always false.}
+          {- If [max] is empty then [Error `Empty_max] is returned.}
+          {- If there is a node [m] in [min] that is not reachable from any
+          element of [max], returns [Error `Disconnected m]. }
           }
 
-          {b Note:} Both [min] and [max] are subsets of [c].*)
+          {b Note:} If [Ok n] is returned then both [min] and [max] are
+          included in [c].*)
```

Comment:
```suggestion
          included in [c].*)
```
After a recent change, `c` is now an unbound variable here :upside_down_face: Perhaps you meant to change it to `n` as well?